dbms/src/TestUtils/mockExecutor.cpp

```diff
@@ -40,7 +41,7 @@ ASTPtr buildOrderByItemList(MockOrderByItems order_by_items)
     size_t i = 0;
     for (auto item : order_by_items)
     {
-        int direction = item.second ? 1 : -1; // todo
+        int direction = item.second ? -1 : 1;
```

Comment:
why?
`1 : -1` change to `1 ： -1`

Reply:
1 for ASC, -1 for DESC, so the previous implementation is wrong.